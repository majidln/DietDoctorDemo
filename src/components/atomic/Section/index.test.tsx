import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import {Section} from './index';

it('Section render correctly', () => {
  renderer.create(<Section title="open up" />);
});

it('Section Snapshot renders correctly', () => {
  const tree = renderer.create(<Section title="open down" />).toJSON();
  expect(tree).toMatchSnapshot();
});
