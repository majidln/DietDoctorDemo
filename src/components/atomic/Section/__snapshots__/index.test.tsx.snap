// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Snapshot renders correctly 1`] = `
<View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexDirection": "row",
        "height": 30,
        "justifyContent": "space-between",
        "opacity": 1,
      }
    }
    testID="sectionButton"
  >
    <Text
      style={
        Object {
          "fontSize": 18,
          "fontWeight": "700",
        }
      }
    >
      open down
    </Text>
    <Text>
      Show Detail
    </Text>
  </View>
</View>
`;
